src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/AddColumns_SupportIdentifiers.txt

```diff
@@ -0,0 +1,31 @@
+﻿#SETUP: SupportIdentifiers
+
+>> AddColumns([1,2,3,4,5], Double, Value * 2)
```

Comment:
tests:
-  what if we add a column name that already exists? (is that an error) 

- binding:

>> With( { b : 15 },
  AddColumns([1,2], 
     b ,   <-- this is the new column name
     Value * b) <-- this 'b' binds against With's scope. 
)

[15, 30]